Uplift/Views/MainView.swift

```diff
@@ -21,6 +21,9 @@ struct MainView: View {
 
     var body: some View {
         ZStack {
+            Color.white
```

Comment:
Is it possible to set the background color of the ZStack to white rather than making Color.white be a view?